src/connector/src/parser/debezium/avro_parser.rs

```diff
@@ -204,6 +204,7 @@ mod tests {
         let inner_shema_str = r#"{
     "type": "record",
     "name": "Value",
+    "namespace": "dbserver1.inventory.customers",
```

Comment:
This is as reasonable as `select v from t` being equivalent to `select t.v from t`. But I have not checked how we are handling namespaces in other places.